app/public/views/patient-form.html

```diff
@@ -0,0 +1,84 @@
+<div class="container">
+  <div class="row">
+    <div class="col-xs-12 col-sm-12 col-md-8 col-lg-8 col-xs-offset-0 col-sm-offset-0 col-md-offset-2 col-lg-offset-2 toppad" >
+      <H3>Informations sur le patient</H3>
+      <div class="panel panel-info">
+        <div class="panel-heading">
+          <h3 class="panel-title"><td></h3>
+        </div>
+        <div class="panel-body">
+          <div class="row">
+            <div class="col-md-3 col-lg-3 " align="center"> <img alt="User Pic" src="\img\male-picture.png" class="img-circle img-responsive"> </div>
+            <div class=" col-md-9 col-lg-9 ">
+              <table class="table table-user-information">
+                <tbody>
+                  <tr>
+                    <td>Prénom:</td>
+                    <td><input type="text" class="form-control" value="" placeholder="Prénom" ng-model='user.name' required></td>
+                  </tr>
+                  <tr>
+                    <td>Nom:</td>
+                    <td><input type="text" class="form-control" value="" placeholder="Nom" ng-model='user.lastName' required></td>
+                  </tr>
+                  <tr>
+                    <td>ID:</td>
+                    <td><input type="number" class="form-control" value="" placeholder="Identifiant" ng-model='user.id' required></td>
+                  </tr>
+                  <tr>
+                    <td>Date de naissance (Âge: {{patient.age}}):</td>
+                    <td><input type="date" class="form-control" value="" placeholder="Date de naissance" ng-model='user.dateNaissance' required></td>
```

Comment:
ng-model='user.dateNaissance' : on semble coder en anglais, keep it this way ;) Le même commentaire s'applique partout.